hack/update-bazel.sh

```diff
@@ -22,16 +22,15 @@ KOPS_ROOT=$(git rev-parse --show-toplevel)
 cd ${KOPS_ROOT}
 TMP_GOPATH=$(mktemp -d)
 
+ln -s ${KOPS_ROOT}/vendor/ ${TMP_GOPATH}/src
```

Comment:
maybe we should delete this link in a `trap ... EXIT` ?

Reply:
@stevekuznetsov I blame you for showing me that bash basically has `defer` :rofl:

Reply:
it's worse, though, since you can only have one trap handler per signal, which is how you end up with things like [`kube::util::trap_add`](https://github.com/kubernetes/kubernetes/blob/8f8201691ee9c15ddb8570c7de6916c3cff4c610/hack/lib/util.sh#L45-L68).

Reply:
well and `defer` is function scoped, so you'd have to put each trap in a subshell, but you can so... obviously we just put all our logic in bash :joy:

Reply:
also that util is kinda horrible and amazing :joy:

Reply:
pls no :(